test/functional/tests/fault_injection/test_cache_insert_error.py

```diff
@@ -81,21 +103,58 @@ def test_cache_insert_error(cache_mode, cache_line_size):
             fio_cmd.target(core_device).verify_only().run()
 
 
-@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.parametrizex("cache_mode", [CacheMode.WT, CacheMode.WA, CacheMode.PT])
+@pytest.mark.parametrizex("block_size", [start_size, Size(4, Unit.KibiByte), stop_size])
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_verify_cache_insert(cache_mode, block_size):
+    """
+    title: Verify contents of cache on insert in no lazy modes
+    description: |
+      Verify I/O operation on non-lazy-writes cache modes
```

Comment:
Verify contents of primary storage after writes to cache with underlaying error device in non-lazy cache mode and check that all IO requests succeeded